packages/dx-scheduler-core/src/plugins/appointment-form/helpers.js

```diff
@@ -0,0 +1,53 @@
+import moment from 'moment';
+
+export const changeAppointmentField = (
+  condition, actionWhileTrue, actionWhileFalse, setAppointmentField,
+) => (nextValue) => {
+  if (condition) {
+    actionWhileTrue({ change: setAppointmentField({}, nextValue) });
+  } else {
+    actionWhileFalse({ change: setAppointmentField({}, nextValue) });
+  }
+};
+
+export const conditionalActionCall = (action, payload) => {
```

Comment:
I suggest renaming this function to `invokeActionIfExists` or `callActionIfExists`. The idea is using a verb as funciton name.